tensorflow/inference/docker/2.2.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,105 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI" 
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+
+ARG PYTHON=python3.7
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.7.7
+ARG OPENSSL_VERSION=1.1.1g
+ARG TFS_SHORT_VERSION=2.2
+ARG TFS_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/serving/r2.2_aws/20200606-011120/cpu/py37/tensorflow_model_server
+
+# See http://bugs.python.org/issue19846
+ENV LANG=C.UTF-8
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV SAGEMAKER_TFS_VERSION="${TFS_SHORT_VERSION}"
+ENV PATH="$PATH:/sagemaker"
+ENV LD_LIBRARY_PATH='/usr/local/lib:$LD_LIBRARY_PATH'
+ENV MODEL_BASE_PATH=/models
+# The only required piece is the model name in order to differentiate endpoints
+ENV MODEL_NAME=model
+ENV DEBIAN_FRONTEND=noninteractive
+
+# nginx + njs 
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends \
+    curl \
+    gnupg2 \
+    ca-certificates \
+    git \
+    wget \
+    vim \
+    build-essential \
+    zlib1g-dev \
+ && curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - \ 
+ && echo 'deb http://nginx.org/packages/ubuntu/ bionic nginx' >> /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get -y install --no-install-recommends \
+    nginx \
+    nginx-module-njs \
+    python3 \
```

Comment:
Please change this logic so that it download py37 instead

Reply:
You can refer the training TF2.2 dockerfile on how the python 37 is download with help of argument